app/views/locations/edit.erb

```diff
@@ -10,6 +10,9 @@
       <%= f.govuk_text_field :address,
                              hint: { text: "For example, 10 High Street, London" } %>
       <%= f.govuk_text_field :postcode %>
+      <%= f.govuk_collection_select :organisation_id, Organisation.all, :id, :name,
```

Comment:
This should probably be limited to all orgs the user is a member of and not all orgs. This should also be enforced in the controller

Reply:
maybe make an exception for a super admin?